ms2query/train_ms2query_nn.py

```diff
@@ -0,0 +1,105 @@
+import os
+import pickle
+import pandas as pd
+from typing import Union, List, Tuple, Dict
+from tensorflow.keras.models import Sequential
+from tensorflow.keras.callbacks import EarlyStopping, ModelCheckpoint
+from tensorflow.keras.layers import Dense
+from matplotlib import pyplot as plt
+
+
+def train_ms2query_nn(x_train: pd.DataFrame, y_train: pd.DataFrame,
```

Comment:
I would split this in two functions: `create_ms2query_nn` and `train_ms2query_nn` because with the current title I would not expect the network creation to be part of it.

Reply:
Yes, I agree, I changed this. I think this function still will have quite some changes/ additions later on. (for that reason I also did not yet create a test function)